go/responses/types.go

```diff
@@ -0,0 +1,49 @@
+package responses
+
+import (
+	"fmt"
+
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/hexutil"
+	"github.com/ethereum/go-ethereum/core/types"
+)
+
+type Encryptor func([]byte) ([]byte, error)
```

Comment:
comment

Reply:
isn't there like only one type of "encryptor", which is a "ViewingKeyEncryptor(vk)"?